src/setup.py

```diff
@@ -27,6 +27,9 @@
 pkg_info = os.path.join(package_dir, "PKG-INFO")
 in_source_package = os.path.isfile(pkg_info)
 if in_source_package:
+    if (sys.version_info[0] == 3 and sys.version_info[1] < 5) or \
+       (sys.version_info[0] == 2 and sys.version_info[1] < 7):
+        raise Exception("Frida only provides prebuilts for python 2.7 and 3.5")
```

Comment:
This is a python 3 thing (implied chained exceptions). I can suppress it with `raise Exception(...) from None`, but then I need to check for the python version.